### DISCUSSION TITLE
Building linux native executables without GraalVM installed

### DISCUSSION THREAD

**Discussion by zero11-0203** [micronaut-projects/micronaut-core#10048] [Q&A]
Hi,

Does micronaut support building linux native exetuables without GraalVM installed like quarkus does?

https://quarkus.io/guides/building-native-image#tip-quarkus-native-remote-container-build

**Comment by graemerocher** [comment]
yup... with Gradle `./gradlew dockerBuildNative` or with maven `mvn package -Dpackaging=docker-native`

**Reply by zero11-0203** [reply]
This will produce a docker image containing the native executable. Is there a way I can get the executable directly on my host machine? I can copy it from a running docker container, but it's kind of troublesome.
Thanks.
